### ISSUE TITLE
Asset creating process loads forever with no result

### ISSUE THREAD

**Issue by tector** [iotaledger/IOTA-2.0-DevNet-wallet#29]
I wanted to create my own asset with the wallet (v0.7.0).
I could enter Name, Symbol and Amount.. then i clicked OK and the process seems to start.
However: The loading animation runs forever without any result.

**Comment by tector** [comment]
today i tried it again.. now i can see an errror:
![grafik](https://user-images.githubusercontent.com/205455/120615236-5bb9c700-c458-11eb-8049-d83b4f4d0819.png)

**Comment by renewid** [comment]
yes, this is what i also got while trying to create a new colored token. got enough IOTAs to colorize and the wallet was connected.

**Comment by renewid** [comment]
![grafik](https://user-images.githubusercontent.com/65234719/120929977-94bd9a00-c6eb-11eb-9774-c0fa77da05ae.png)
